index.js

```diff
@@ -36,9 +38,11 @@ module.exports = (api, options) => {
 
   const cordovaRun = () => {
     // cordova run platform
-    info('executing "cordova run ' + platform + '"...')
+    // uses build for browser since run would trigger cordovaServe
+    const cmd = platform === 'browser' ? 'build' : 'run'
+    info('executing "cordova ' + cmd + ' ' + platform + '"...')
```

Comment:
To be honest kept the same style, but it would be nice to use string interpolation:

```js
info(`executing "cordova ${cmd} ${platform}"...`)
```